pkg/security/context.go

```diff
@@ -0,0 +1,22 @@
+package security
+
+type Context interface {
+	GetResponseOboToken() string
+	SetResponseOboToken(token string)
+}
+
+type contextImpl struct {
+	responseOboToken string
+}
+
+func NewContext() Context {
```

Comment:
probably we can get rid of this structure. Just simply put two strings into context via context.WithValue()

Reply:
We need structure to allow `SecurityService` see `responseOboToken` that had been set by `EndpointService`